kolibri/plugins/learn/assets/src/modules/topicsTree/handlers.js

```diff
@@ -1,37 +1,27 @@
+import { get } from '@vueuse/core';
 import { ContentNodeResource, ContentNodeProgressResource } from 'kolibri.resources';
 import samePageCheckGenerator from 'kolibri.utils.samePageCheckGenerator';
-import ConditionalPromise from 'kolibri.lib.conditionalPromise';
 import router from 'kolibri.coreVue.router';
 import { PageNames } from '../../constants';
+import useChannels from '../../composables/useChannels';
 import { _collectionState, normalizeContentNode, contentState } from '../coreLearn/utils';
 
-export function showTopicsChannel(store, id) {
-  return store.dispatch('loading').then(() => {
-    store.commit('SET_PAGE_NAME', PageNames.TOPICS_CHANNEL);
-    return showTopicsTopic(store, { id, isRoot: true });
-  });
-}
+const { channelsMap } = useChannels();
 
 export function showTopicsContent(store, id) {
-  store.commit('SET_EMPTY_LOGGING_STATE');
   store.commit('CORE_SET_PAGE_LOADING', true);
   store.commit('SET_PAGE_NAME', PageNames.TOPICS_CONTENT);
 
-  const promises = [
-    ContentNodeResource.fetchModel({ id }),
-    ContentNodeResource.fetchNextContent(id),
-    store.dispatch('setChannelInfo'),
-  ];
-  ConditionalPromise.all(promises).only(
+  ContentNodeResource.fetchModel({ id }).only(
     samePageCheckGenerator(store),
-    ([content, nextContent]) => {
```

Comment:
I can't totally tell if this is a refactor, or a removal of "next content". Does any of @nucleogenesis 's side panel work depend on this for the topics/lesson overview side panel?

Reply:
My recollection was that this was being independently fetched in the side panel, and so it was now superfluous here.

I think that is still correct, as I see the recommendations being loaded here: https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/plugins/learn/assets/src/views/BrowseResourceMetadata.vue#L298